python/mxnet/ndarray_doc.py

```diff
@@ -77,6 +77,15 @@ class BroadcastToDoc(NDArrayDoc):
     (2L, 2L, 2L, 3L)
     """
 
+class CustomDoc(NDArrayDoc):
+    """
+    Example
+    -------
+    Applies a custom operator named `my_custom_operator` to `input`.
+
+    >>> output = mx.symbol.Custom(op_type='my_custom_operator', data=input)
```

Comment:
would it help to show a small sample custom operator and how it works.
I don't see much value with just this since you have already referenced the tutorial